From 626ad760ecdfd1ba865e68e8a179566dae013244 Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Wed, 8 Apr 2015 15:55:53 -0700
Subject: [PATCH] dma: dw: enable Cougar Mountain LPDMA

Add Cougar Mountain's LPDMA ACPI ID(80862B86, 80862BC0) to the
mapping table.

Signed-off-by: Brett T. Warden <brett.t.warden@intel.com>
---
 drivers/acpi/acpi_lpss.c  |    4 ++--
 drivers/dma/dw/platform.c |    2 ++
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/drivers/acpi/acpi_lpss.c b/drivers/acpi/acpi_lpss.c
index 29ef29b..c17b676 100644
--- a/drivers/acpi/acpi_lpss.c
+++ b/drivers/acpi/acpi_lpss.c
@@ -202,8 +202,8 @@ static const struct acpi_device_id acpi_lpss_device_ids[] = {
 	/* Cougar Mountain LPSS devices */
 	{ "80862BC1", (unsigned long)&cht_i2c_dev_desc },
 	{ "80862B8A", (unsigned long)&byt_uart_dev_desc },
-	{ "80862B86", }, /* CGM DMA1 */
-	{ "80862BC0", }, /* CGM DMA2 */
+	{ "80862B86", (unsigned long)&lpss_dma_desc },
+	{ "80862BC0", (unsigned long)&lpss_dma_desc },
 	{ "80862B8E", (unsigned long)&byt_spi_dev_desc },
 
 	{ }
diff --git a/drivers/dma/dw/platform.c b/drivers/dma/dw/platform.c
index e40c8da..c2c3010 100644
--- a/drivers/dma/dw/platform.c
+++ b/drivers/dma/dw/platform.c
@@ -256,6 +256,8 @@ static const struct acpi_device_id dw_dma_acpi_id_table[] = {
 	{ "INTL9C60", 0 },
 	{ "80862286", 0 },
 	{ "808622C0", 0 },
+	{ "80862B86", 0 },
+	{ "80862BC0", 0 },
 	{ }
 };
 MODULE_DEVICE_TABLE(acpi, dw_dma_acpi_id_table);
-- 
1.7.9.5

